### PR TITLE
PKX-241 associate new fields to users and create a group for training managers

### PR DESCRIPTION
**Pull request by Faizan-at-Edly**

### CODE REVIEW

File: common/djangoapps/student/models.py [hunk 1]
```diff
@@ -448,6 +448,9 @@ class Meta(object):
     user = models.OneToOneField(User, unique=True, db_index=True, related_name='profile', on_delete=models.CASCADE)
     name = models.CharField(blank=True, max_length=255, db_index=True)
 
+    # EmployeeID can be null
```
Comment (mubtadaali) [inline common/djangoapps/student/models.py:451]: no need for this comment.

File: openedx/features/pakx/lms/pakx_admin_app/constants.py [hunk 2]
```diff
@@ -0,0 +1 @@
+Group_Training_Managers = "Training Manager"
```
Comment (mubtadaali) [inline openedx/features/pakx/lms/pakx_admin_app/constants.py:1]: constants should be defined in all capital letters with underscores separating words. 
please follow pep8

File: openedx/features/pakx/lms/pakx_admin_app/views.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.response import Response
+from .permissions import CanAccessPakXAdminPanel
+
+
+@api_view(('GET',))
+@permission_classes([CanAccessPakXAdminPanel])
+def temp(request):
```
Comment (mubtadaali) [inline openedx/features/pakx/lms/pakx_admin_app/views.py:8]: this should be part of this PR. PR should not contain any dev testing code

**Review by mubtadaali — CHANGES_REQUESTED** [review]

File: openedx/features/pakx/lms/pakx_admin_app/views.py (hunk 3 above)
Comment (Faizan-at-Edly) [reply openedx/features/pakx/lms/pakx_admin_app/views.py]: Done!

File: openedx/features/pakx/lms/pakx_admin_app/constants.py (hunk 2 above)
Comment (Faizan-at-Edly) [reply openedx/features/pakx/lms/pakx_admin_app/constants.py]: Done!

File: common/djangoapps/student/models.py (hunk 1 above)
Comment (Faizan-at-Edly) [reply common/djangoapps/student/models.py]: Done!

File: openedx/features/pakx/lms/pakx_admin_app/apps.py [hunk 4]
```diff
@@ -3,4 +3,5 @@
 
 class PakxAdminAppConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
-    name = 'pakx_admin_app'
+    name = 'openedx.features.pakx.lms.pakx_admin_app'
+    verbose_name = 'PakistanX Admin Panel App'
```
Comment (mubtadaali) [inline openedx/features/pakx/lms/pakx_admin_app/apps.py:7]: end of line missing

**Review by mubtadaali — APPROVED** [review]
LGTM

File: openedx/features/pakx/lms/pakx_admin_app/apps.py (hunk 4 above)
Comment (Faizan-at-Edly) [reply openedx/features/pakx/lms/pakx_admin_app/apps.py]: Done!

**Review by mubtadaali — APPROVED** [review]

**Pull request merged**